js/src/widgets/editorPanel.js

```diff
@@ -68,6 +68,8 @@
 
               return false;
             });
+            var annos = { data: state.annotations };
+            jQuery.publish('annotationsListFiltered' + this.windowId, annos);
```

Comment:
Should the editor tab be publishing this? Or should the annotationsTab be publishing it?